GliaWidgets/Sources/View/Common/Alert/AlertStyle.RemoteConfig.swift

```diff
@@ -0,0 +1,117 @@
+import UIKit
+
+// MARK: - Public
```

Comment:
Would it be `Public` or `Internal`?

Reply:
fixed [d2cf4a7](https://github.com/salemove/ios-sdk-widgets/pull/887/commits/d2cf4a7c8c63bcb2a3064f0ed6f678ec4995ac7f)